### DISCUSSION TITLE
建立账号的时候邮箱格式输错了，可以帮忙改过来吗

### DISCUSSION THREAD

**Discussion by Asatsukisora** [getlantern/lantern-forum-cn#591] [Q&A] [answered]
之前创号的时候输入邮箱 不小心末尾少打了m，邮箱从ayayashana@gmail.com变成了ayayashana@gmail.co 虽然能正常使用但是每次授权新设备都得要用授权码……所以请问一下方不方便把邮箱改正@Mike777Khoo 麻烦了

**Comment by MaxMKMax** [comment]
@Asatsukisora 您好. 好的, 没问题. 我帮您提交问题上去处理.

**Comment by MaxMKMax** [comment, marked as answer]
@Asatsukisora 您好. 邮件已经帮您改好了.

**Reply by Asatsukisora** [reply]
确认过了 也能正常接收到邮件 谢谢您！
